build_tools/docker/manage_images.py

```diff
@@ -94,8 +93,7 @@ def parse_arguments():
       type=str,
       default='latest',
       help='Tag for the images to build. Defaults to `latest` (which is good '
-      'for testing changes in a PR). Use `prod` to update the images that the '
-      'CI caches.')
+      'for testing changes in a PR). The `prod` tag will not work with --push.')
```

Comment:
It seems a bit weird to have a totally separate script just for prod. I think the utility of the tag argument is also basically zero without the prod tag, so if you do keep it as separate scripts I think it might be better to get rid of it entirely.

Reply:
I wanted to make this a separate script so that the API and control flow for `manage_images.py` wouldn't get any more complicated, and so that the functionality of pushing to prod is firewalled from other tasks related to managing images that can't negatively affect others.

Happy to remove the `--tag` flag if you're okay with two scripts :)

Reply:
Ok SGTM